### PR TITLE
Update Project and Documentation URLS

### PR DESCRIPTION
**Pull request by ph1l**
did some repo-wide search and replaces to:

* update project url to new github organization
* update my email address and copyright lines in software

### CODE REVIEW

**Comment by ph1l** [comment]
Also, @patfreeman, I'm pretty sure the code counts as the original license anyway, but you should probably add your copyright to the project too... in the files you've edited (at leasst substantially,) and in the debian/copyright, I think.

File: doc/GPL-STUB [hunk 1]
```diff
@@ -17,5 +17,5 @@
 #
 #
 #########################################################################
-#       Copyright 2011 Philip Freeman <philip.freeman@gmail.com>
+#            Copyright 2018 Author Name <email@address>
```
Comment (patfreeman) [inline doc/GPL-STUB]: email@address?

File: doc/GPL-STUB.slashes [hunk 2]
```diff
@@ -18,6 +18,6 @@
 //
 //
 //########################################################################
-//       Copyright 2011 Philip Freeman <philip.freeman@gmail.com>
+//           Copyright 2018 Author Name <email@address>
```
Comment (patfreeman) [inline doc/GPL-STUB.slashes]: email@address?

File: doc/GPL-STUB (hunk 1 above)
Comment (ph1l) [reply doc/GPL-STUB]: removed

File: doc/GPL-STUB.slashes (hunk 2 above)
Comment (ph1l) [reply doc/GPL-STUB.slashes]: removed stub

**Pull request merged**